### PR TITLE
feat: add first implementation

### PR DESCRIPTION
**Pull request by mdujava**
First mockups of the app.
Sliders needed update of the "@patternfly/react-core" to include fix for https://github.com/patternfly/patternfly-react/issues/5833

image

Signed-off-by: Matej Dujava <mdujava@kocurkovo.cz>

### CODE REVIEW

**Comment by renanrodrigo** [comment]
Thanks @mdujava ! Will drop a review ASAP (:

File: src/components/mainScreen/MainScreen.tsx [hunk 1]
```diff
@@ -1,7 +1,155 @@
...
+  const res = [
+    ['1 day', '2 hours', '30 minutes', '4 minutes', '1 minute', '5 seconds'],
+    ['5 days', '12 hours', '2 hours', '21 minutes', '5 minutes', '25 seconds'],
+    ['4 weeks', '3 days', '12 hours', '2 hours', '30 minutes', '2 minutes'],
+    ['8 weeks', '6 days', '1 day', '4 hours', '1 hour', '5 minutes'],
+    ['9 months', '4 weeks', '6 days', '21 hours', '5 hours', '25 minutes'],
+    ['NaN', '6 months', '5 weeks', '5 days', '1 day', '2 hours'],
+    ['NaN', '10 months', '2 months', '10 days', '2 days', '5 hour'],
+    ['NaN', 'NaN', 'NaN', '2 months', '2 weeks', '1 day'],
+    ['NaN', 'NaN', 'NaN', 'NaN', '8 weeks', '5 days'],
```
Comment (renanrodrigo) [inline src/components/mainScreen/MainScreen.tsx:60]: Why use NaN here? I think we could have a better value for those cases.
The fixed matrix is good for the first version, but in the future we might do it dynamically

File: src/components/mainScreen/MainScreen.tsx [hunk 2]
```diff
@@ -1,7 +1,155 @@
...
+  ];
+
+  const res = [
+    ['1 day', '2 hours', '30 minutes', '4 minutes', '1 minute', '5 seconds'],
+    ['5 days', '12 hours', '2 hours', '21 minutes', '5 minutes', '25 seconds'],
+    ['4 weeks', '3 days', '12 hours', '2 hours', '30 minutes', '2 minutes'],
+    ['8 weeks', '6 days', '1 day', '4 hours', '1 hour', '5 minutes'],
+    ['9 months', '4 weeks', '6 days', '21 hours', '5 hours', '25 minutes'],
+    ['NaN', '6 months', '5 weeks', '5 days', '1 day', '2 hours'],
+    ['NaN', '10 months', '2 months', '10 days', '2 days', '5 hour'],
```
Comment (renanrodrigo) [inline src/components/mainScreen/MainScreen.tsx]: 5 hours

File: src/components/mainScreen/MainScreen.tsx [hunk 3]
```diff
@@ -1,7 +1,155 @@
...
+                  showTicks
+                  max={8}
+                />
+              </StackItem>
+              <StackItem>
+                <Split>
+                  <SplitItem isFilled />
+                  <SplitItem>
+                    {' '}
+                    You should not spend more time then: {automation}{' '}
```
Comment (renanrodrigo) [inline src/components/mainScreen/MainScreen.tsx]: s/then/than

**Review by renanrodrigo — CHANGES_REQUESTED** [review]

File: src/components/mainScreen/MainScreen.tsx (hunk 1 above)
Comment (mdujava) [reply src/components/mainScreen/MainScreen.tsx:60]: Yes, this is just first implementation, just converting the table from the comics into the code. In future we can dynamically calculate times.

**Comment by renanrodrigo** [comment]
Thanks, @mdujava  - we have a great prototype now :eyes:

**Pull request merged**